packages/fluentui/react-northstar/src/components/RadioGroup/RadioGroupItem.tsx

```diff
@@ -118,14 +118,22 @@ export const RadioGroupItem: ComponentWithAs<'div', RadioGroupItemProps> &
     initialValue: false,
   });
 
+  const prevChecked = React.useRef<boolean>(checked);
+
   const handleClick = (e: React.MouseEvent | React.KeyboardEvent) => {
     _.invoke(props, 'onClick', e, props);
     setChecked(prevChecked => {
-      _.invoke(props, 'onChange', undefined, { ...props, checked: !prevChecked });
       return !prevChecked;
     });
   };
 
+  React.useEffect(() => {
+    if (prevChecked.current !== checked) {
+      _.invoke(props, 'onChange', undefined, { ...props, checked });
```

Comment:
So I think maybe a TODO to make a breaking change and make `onChange` behave natively (only on selection) might make sense, I think @layershifter  would also agree

Reply:
Added

Reply:
LGTM